system/Common.php

```diff
@@ -563,6 +563,24 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('solidus')) {
+    /**
+     * Generates the solidus character (`/`) depending on the HTML5 compatibility flag in `Config\DocTypes`
+     *
+     * @internal
+     */
+    function solidus(?bool $html5 = null): string
```

Comment:
Why did you add this parameter? This internal function should only operate on the `config('DocTypes')->html5` property. For testing, you can mock the config and set the value of `$html5` to false.